src/plot_utils.py

```diff
@@ -88,6 +88,18 @@ def heatmap(bdb, bql, vmin=None, vmax=None, row_ordering=None,
         Active BayesDB instance.
     bql : str
         The BQL to run and plot. Must be a PAIRWISE BQL query.
+    **kwargs : dict
+        Passed to df_heatmap
+    Returns
+    -------
+    clustermap: seaborn.clustermap
+    """
+    df = bqlu.cursor_to_df(bdb.execute(bql))
+    df.fillna(0, inplace=True)
+    return df_heatmap(df, **kwargs)
+
+def df_heatmap(df, vmin=None, vmax=None, row_ordering=None, col_ordering=None):
```

Comment:
As I started looking farther down, and looking at __init__.py, I decided you were right: the only thing df_heatmap added over and above zmatrix was a better default value for the clustermap_kws, so I just added that to zmatrix, and will use that from outside as bdbcontrib.plot_utils.zmatrix, which is fine from quickstart.